src/shared.ts

```diff
@@ -102,6 +117,14 @@ const createWrapper = (stripeConnect: any) => {
           }
         }
       });
+      const oldCreate = stripeConnectInstance.create.bind(
+        stripeConnectInstance
+      );
+      stripeConnectInstance.create = (tagName: ConnectElementTagName) => {
+        const htmlName = componentNameMapping[tagName];
+        return oldCreate(htmlName);
```

Comment:
Should we add this:
```suggestion
        if (!htmlName) {
          throw new Error(`The element ${tagName} is not supported. Please specify one of these ${htmlTagNames.join(', ')}`);
        }
        return oldCreate(htmlName);
```

Reply:
Actually - disregard my suggestion. It should be something like this instead:

```
        if (!htmlName) {
          htmlName = tagName;
        }
```

That way we can still specify private components... this is making me think this should perhaps go in `submerchant-surfaces`, however the typescript types consistency argument still applies, so I'd say this is ok!